packages/oncokb-frontend-commons/src/components/OncoKbCard.tsx

```diff
@@ -0,0 +1,101 @@
+import { IndicatorQueryResp } from 'oncokb-ts-api-client';
+import * as React from 'react';
+
+import { ICache } from '../model/SimpleCache';
+
+import oncoKbLogoImgSrc from 'oncokb-styles/dist/images/logo/oncokb.svg';
+import mainStyles from './main.module.scss';
+import { OncoKbCardTitle } from './OncoKbCardTitle';
+import { OncoKbCardBody } from './OncoKbCardBody';
+import { OncoKbCardDataType } from 'cbioportal-utils';
+
+export type OncoKbCardProps = {
+    type: OncoKbCardDataType;
+    hugoSymbol: string;
+    geneNotExist: boolean;
+    isCancerGene: boolean;
+    usingPublicOncoKbInstance: boolean;
+    pmidData: ICache;
+    indicator?: IndicatorQueryResp;
+    displayHighestLevelInTabTitle?: boolean;
+    handleFeedbackOpen?: React.EventHandler<any>;
+};
+
+export const OncoKbCard: React.FunctionComponent<OncoKbCardProps> = (
+    props: OncoKbCardProps
+) => {
+    function oncokbLinkOut() {
+        let link: string | undefined = undefined;
+        if (props.hugoSymbol) {
+            link = `https://www.oncokb.org/gene/${props.hugoSymbol}`;
+            if (
+                !props.geneNotExist &&
+                props.indicator &&
+                props.indicator.variantExist
+            ) {
+                link = `${link}/${props.indicator.query.alteration}`;
+            }
+        }
+        return link;
+    }
+
+    const oncokbLogo = (
+        <img
+            src={oncoKbLogoImgSrc}
```

Comment:
i would be a little suprrised if this worked because how would it resolve the image path at runtime.  usually we use the webpack image loader.  where you set src to require("pathToImage"). for small images, webpack will inline the src of the image.    let me know if i'm wrong and it actually works now.

Reply:
@alisman I think rollup is doing a similar thing with this plugin: https://github.com/cBioPortal/cbioportal-frontend/blob/7578f43b14989fd332c2b9038afb3512c722c4d7/packages/config/rollup.config.js#L42-L46

This may not be the best solution for larger images because it inlines everything regardless of its size. But ideally, we shouldn't have larger images in our packages.

Reply:
This does currently work but I can change it if you prefer

Reply:
if it works it works!     you can just confirm that it inlines the image?  like, in the browser if you inspect the image, you should see a big long data uri